test/asyncio_tests/test_asyncio_ssl.py

```diff
@@ -195,3 +195,35 @@ async def test_cert_ssl_validation_hostname_fail(self):
                 )
 
                 await client.db.collection.find_one()
+
+    @asyncio_test
+    async def test_shared_client(self):
+        # Start AWS Lambda Example 1
+        import os
+
+        from motor import MotorClient
+
+        client = MotorClient(host=os.environ["MONGODB_URI"])
+
+        def lambda_handler(event, context):
+            return client.db.command("ping")
```

Comment:
`command` is async, so these functions should be run as `asyncio.run()`, or `IOLoop.run_sync` for `tornado`.